sites/cheerpx/src/content/docs/12-reference/00-CheerpX-Linux-create.md

```diff
@@ -79,3 +79,173 @@ networkInterface?: NetworkInterface;
 This option configures network settings, which allows CheerpX to communicate over networks. For more detailed information about how CheerpX handles networking, including the use of Tailscale and overcoming browser limitations, see the [Networking](/docs/guides/Networking) guide.
 
 [Promise]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
+
+### `authKey`
+
+```ts
+authKey?: string;
+```
+
+The `authKey` is an optional string used for authentication. It is usually taken from the URL, allowing the CheerpX Linux environment to be set up automatically without the user needing to enter it manually.
+
+**Example of usage:**
+
+```js
+let authKey = undefined;
+if (browser) {
+	let params = new URLSearchParams("?" + window.location.hash.substr(1));
+	authKey = params.get("authKey") || undefined;
+}
+const networkInterface = { authKey };
+```
+
+**Explanation**
+
+1. The `authKey` is taken from the URL, allowing the application to automatically use the authentication key provided by the browser.
+
+2. The `authKey` is added to the `networkInterface` object, ensuring the application can access the authentication data whenever needed in the CheerpX environment.
+
+### `controlUrl`
+
+```ts
+controlUrl?: string;
+```
+
+The `controlUrl` is an optional string parameter representing a URL used to manage or monitor the connection. It is typically fetched from the URL parameters and can be utilized for dashboard or configuration purposes.
```

Comment:
Same as before. Simplify this in general. The purpose of this option is to select the self-hosted server, all the rest is noise.

In general prefer explaining what is the purpose of options from the user point of view,. Nobody cares about "the control plain", people just want to self host somehow